### PR TITLE
Adding another "container full" case.

### PR DESCRIPTION
**Pull request by jayRyan24**
Found another on Inoo.

### CODE REVIEW

**Comment by rcuhljr** [comment]
Should this guy be added to combat-trainer's loot?

File: pick.lic [hunk 1]
```diff
@@ -203,10 +203,13 @@ class LockPicker
     end
 
     if @make_pets
-      case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your .* in your ', 'You just can\'t get')
+      case bput("put my #{box} in my #{@settings.picking_pet_box_source}", 'You put your .* in your ', 'You just can\'t get', 'There isn\'t any more')
       when 'You just can\'t get'
```
Comment (rpherbig) [inline pick.lic]: You can combine these two cases with `when 'You just can\'t get', 'There isn\'t any more'`

Comment (rpherbig) [inline pick.lic]: Maybe consider double quoting those strings too

Comment (jayRyan24) [inline pick.lic]: Ah, was figuring you could do that, but I did when case || case, bleh.

**Comment by rpherbig** [comment]
This new string was already in `combat-trainer`'s looting process.

Methinks we need to write some common helpers for get/put/stow...

**Pull request merged**